core/sync_assets.py

```diff
@@ -0,0 +1,57 @@
+import psycopg2
+import sys
+from config_loader import load_config
+from Historic_Crypto import Cryptocurrencies
```

Comment:
this lib? its ok? but in the requirements is registered?

Reply:
Hi, dear Gael, it's imported as Historic-Crypto in requirements.
Please check more info @ https://github.com/David-Woroniuk/Historic_Crypto

Reply:
Thanksss, but in this PR isn't register in requirements:) and for that not installed